lab-jonathan/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "lab-jonathan",
+  "version": "1.0.0",
+  "description": "lab01 project for modular patterns",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
```

Comment:
Don't forget to write a script in here that will enable you to run gulp with NPM